Homework2/docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '2'
+services:
+  postgres:
+      image: postgres:9.6
+
+      environment:
+          - POSTGRES_USER=airflow
+          - POSTGRES_PASSWORD=airflow
+          - POSTGRES_DB=airflow
+      ports:
+          - 5432:5432
+  webserver:
+      image: puckel/docker-airflow:1.8.1
+      restart: always
+      command: 
```

Comment:
Do not specify multiple `command` in the docker-compose file. 
1. The first command will be overwritten by the later one
2. It's not clear for other developers

To have multiple bash commands, do something like this `bash -c "<long string of commands>"` https://github.com/docker/compose/issues/2033